governance/handler.go

```diff
@@ -305,10 +305,8 @@ func (gov *Governance) HandleGovernanceVote(valset istanbul.ValidatorSet, votes
 			logger.Warn("Received Vote was invalid", "number", header.Number, "Validator", gVote.Validator, "key", gVote.Key, "value", gVote.Value)
 		}
 		if number > atomic.LoadUint64(&gov.lastGovernanceStateBlock) {
-			gov.GovernanceVotes = make([]GovernanceVote, len(votes))
-			gov.GovernanceTally = make([]GovernanceTally, len(tally))
-			copy(gov.GovernanceVotes, votes)
-			copy(gov.GovernanceTally, tally)
+			gov.GovernanceVotes.Import(votes)
```

Comment:
The `Import()` method might be confusing. How about this change?

- Import() --> Copy()
- Copy() --> MakeCopy()